flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingMetricCollector.java

```diff
@@ -114,6 +115,7 @@ public CollectedMetricHistory updateMetrics(
         var topology = getJobTopology(ctx, stateStore, jobDetailsInfo);
         var stableTime = jobUpdateTs.plus(conf.get(AutoScalerOptions.STABILIZATION_INTERVAL));
         final boolean isStabilizing = now.isBefore(stableTime);
+        final boolean isForbidding = AutoScalerUtils.inForbidPeriod(conf, now);
```

Comment:
Everywhere the word `Forbid` should be replaced with `Forbidden` or `Blocked` to be correct.

Reply:
fixed